Frontend/android-native/app/src/user/yatri/res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
```

Comment:
@Anuragini-2506 can we please reduce the size of the images being used here. especially , the ones which are more than 15kb

Reply:
done